cli_tools/common/utils/logging/service/daisy_loggable.go

```
@@ -37,6 +37,14 @@ func (w workflowLoggable) GetValue(key string) string {
 	return w.wf.GetSerialConsoleOutputValue(key)
 }
 
+func (w workflowLoggable) GetValueAsBool(key string) bool {
+	v, err := strconv.ParseBool(w.wf.GetSerialConsoleOutputValue(key))
+	if err != nil {
```

Can we propagate this error up the stack?